warden/x/warden/keeper/wasm.go

```diff
@@ -99,18 +100,22 @@ func parseShieldExpression(v json.RawMessage) (*ast.Expression, error) {
 	switch out := out.(type) {
 	case int64:
 		return ast.NewIntegerLiteral(&ast.IntegerLiteral{
-			Value: out,
+			Value: big.NewInt(out).String(),
 		}), nil
 	case int:
 		return ast.NewIntegerLiteral(&ast.IntegerLiteral{
-			Value: int64(out),
+			Value: big.NewInt(int64(out)).String(),
+		}), nil
+	case string:
+		return ast.NewIntegerLiteral(&ast.IntegerLiteral{
+			Value: out,
```

Comment:
```suggestion
```

no, we can't parse all strings as numbers, otherwise we'll lose the actual string values, just delete this case

Reply:
But won't integers input be limited with int64.max in that case? If yes, maybe we can add `case []byte` then